### ISSUE TITLE
Invalid regex message obscures search

### ISSUE THREAD

**Issue by justanotheranonymoususer** [microsoft/vscode#152790]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: May 2022 Recovery 1
- OS Version: Win10

Steps to Reproduce:

![Code_xI7LSuA2Rs](https://user-images.githubusercontent.com/5781692/174890369-e9b936ac-6a41-4426-b8cc-aad3973dfa95.gif)

1. See gif
2. Or: Ctrl+F and quickly Ctrl+V with invalid regex
3. Or: Ctrl+F, invalid regex, Esc, Ctrl+F again

**Comment by justanotheranonymoususer** [comment]
> Thanks for creating this issue! It looks like you may be using an old version of VS Code, the latest stable release is 1.68.1. Please try upgrading to the latest version and checking whether this issue remains.

Another bug report 👆

**Comment by omkarshinge** [comment]
After debugging found the root cause :
- The error message was getting displayed even before the find/search widget takes place after transition/animation which resulted in improper positioning of the error message.
- There's timeout set for validating the find/search widget after reopening it (file : `findWidget.ts`) and it is set to 200ms which causes error message to be shown before the find/search input box is displayed properly.
- Increasing the timeout to 250ms solves the issue and error message is displayed properly below the search input

Have raised PR :  [15013](https://github.com/microsoft/vscode/pull/153013) which resolves the issue by changing the timeout from 200ms to 250ms

**Comment by justanotheranonymoususer** [comment]
Does it fix the case of Ctrl+F and quickly Ctrl+V with invalid regex too?

**Comment by omkarshinge** [comment]
Yes increasing timeout to 300ms fixes the Ctrl+F and quickly Ctrl+V with invalid regex issue too. 

**Tested both the scenarios as mentioned**
1. reopening the invalid input again and again i.e Ctrl+F, invalid regex, Esc, Ctrl+F again
2. Ctrl+F and quickly doing Ctrl+V with invalid regex

Found that 300ms timeout doesn't show any issue. (Checked increasing from 200ms to 250ms but found intermittent issue with it. Increasing it to 300ms is working fine )

**Attaching the GIFs of both the scenarios after making the change**
1. reopening the invalid input again and again 
![existing_invalid](https://user-images.githubusercontent.com/24647310/175390171-1e8c7901-435e-4115-b756-28ca34525b67.gif)
2. Ctrl+F and quickly doing Ctrl+V with invalid regex
![quick_pasting](https://user-images.githubusercontent.com/24647310/175391118-1cc74b67-9582-4b07-bcfa-6ef36db1870f.gif)

**Comment by omkarshinge** [comment]
**One thing to note which is not properly being seen in above attached GIF files regarding the search box error message**

After doing the above fix it looks like the error message tries to reposition itself to proper position within fraction of second (changes from improper position to proper position) [This might be happening due to recalculation of the elements position/height against which error message has to be displayed (here in this case the find/search input box)]

I have attached video recording below in which we can properly see the repositioning of error message while doing Ctrl+F and quickly doing Ctrl+V with invalid regex input (error message shifts from first appearance position to proper position i.e. top to bottom shifting)

https://user-images.githubusercontent.com/24647310/175458192-a7536416-03d2-4422-aef3-d36e46092f84.mp4
